File: C/Program37.c
////////////////////////////////////////////////////////////////////////////////////
//                                                                                //
//  File name :     Program37.c                                                   //
//  Description :   Program to accept 5 intgers from user & display on screen     //
//  Author :        Pallavi Omprakash Malewar                                     //
//  Date :          27/06/2025                                                    //
//                                                                                //
////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

void Display(int *ptr , int iSize)
{
    int iCount = 0 ;

    printf("Elements of the array are : \n");

    for(iCount = 0 ; iCount <= iSize ; iCount++)
    {
        printf("%d\n" , *ptr);
        ptr++;
    }
}


int main()
{
    int Arr[5];
    int iCount = 0;

    printf("Enter the Elements : \n");

    for(iCount = 0 ; iCount < 5 ; iCount++)
    {
        scanf("%d" , &Arr[iCount]);
    }

    Display(Arr , 5);   

    return 0;
}

File: JAVA/Program37.java
//////////////////////////////////////////////////////////////////////////////////////////
//                                                                                      //
//  File name :     Program37.java                                                      //
//  Description :   Used to Print Given Pattern                                         //
//  Author :        Pallavi Omprakash Malewar                                           //
//  Date :          25/08/2025                                                          //
//                                                                                      //
//////////////////////////////////////////////////////////////////////////////////////////



// Input : 4    4
// Output :
/*
    A   A   A   A   
    B   B   B   B
    C   C   C   C
    D   D   D   D
*/
            


import java.util.Scanner;

class Pattern
{
    public void Display(int iRow , int iCol)
    {
        int i = 0, j = 0;
        char ch = 'A';

        for(i = 1, ch = 'A'; i <= iRow ; i++, ch++)
        {
            for(j = 1 ; j <= iCol ; j++)
            {
                System.out.print(ch + "\t");
            }
            System.out.println();
        }
    }
}

class Program37
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0, iValue2 = 0;

        System.out.print("Enter Number of Rows : ");
        iValue1 = sobj.nextInt();

        System.out.print("Enter Number of Columns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue1, iValue2);
    }
}